custom/icds_reports/tests/agg_tests/agg_setup.py

```diff
@@ -239,3 +240,24 @@ def cleanup_misc_agg_tables():
             table = metadata.tables[name]
             delete = table.delete()
             connection.execute(delete)
+
+
+def partition_child_health():
+    engine = connection_manager.get_engine(ICDS_UCR_CITUS_ENGINE_ID)
+    queries = [
+        "ALTER TABLE child_health_monthly RENAME TO child_health_old_partition",
+        "CREATE TABLE child_health_monthly (LIKE child_health_old_partition) PARTITION BY LIST (month)",
+        "SELECT create_distributed_table('child_health_monthly', 'supervisor_id')",
+        "ALTER TABLE child_health_monthly ATTACH PARTITION child_health_old_partition DEFAULT"
+        ]
```

Comment:
@calellowitz how were these changes made on the production DB? I don't see any DB migration.